core/tests/protractor_utils/CommunityDashboardPage.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
Year

Reply:
You mean, change the year from 2018 to 2019, right?

Done!

Reply:
You mean, change the year from 2018 to 2019, right?

Done!

Reply:
yup!